FinSearchUnified/Bundle/SearchBundleFindologic/ResponseParser/Xml21/Filter/FilterMode.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace FinSearchUnified\Bundle\SearchBundleFindologic\ResponseParser\Xml21\Filter;
+
+class FilterMode
```

Comment:
I didn't mean to put this here, but in the [FINDOLOGIC-API](https://github.com/findologic/findologic-api), with the namespace `FINDOLOGIC\Api\Definitions`.